### PR TITLE
[13.0][MIG] Migration of stock_orderpoint_move_link

### PR DESCRIPTION
**Pull request by guewen**

### CODE REVIEW

File: stock_orderpoint_move_link/models/stock_warehouse_orderpoint.py [hunk 1]
```diff
@@ -2,13 +2,12 @@
 #   (http://www.eficent.com)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
```
Comment (nikul-serpentcs) [inline stock_orderpoint_move_link/models/stock_warehouse_orderpoint.py:3]: @guewen Check License `AGPL` ->` LGPL`

**Review by LoisRForgeFlow — APPROVED** [review]
Code review LGTM

**Review by JoanSForgeFlow — APPROVED** [review]
Tested on local, LGTM

**Comment by JoanSForgeFlow** [comment]
If possible, would you mind to change "Eficent" by "ForgeFlow" in __manifest__.py declaration?
Thank you

**Comment by LoisRForgeFlow** [comment]
Let's merge as it is, we can always fix copyrights later, it is not a big deal.

/ocabot merge

**Review by AdriaGForgeFlow — APPROVED** [review]

**Pull request merged**